raiden/encoding/messages.py

```diff
@@ -42,6 +42,7 @@ def make_message(message, **attrs):
 
 nonce = make_field('nonce', 8, '8s', integer(0, UINT64_MAX))
 payment_identifier = make_field('payment_identifier', 8, '8s', integer(0, UINT64_MAX))
+chain_id = make_field('chain_id', 8, '8s', integer(0, UINT64_MAX))
```

Comment:
`chain_id` is defined as a [`uint256`](https://github.com/raiden-network/raiden-contracts/blob/2ab91aa4b062d88a124ce050551e2f4b8cb0da57/raiden_contracts/contracts/TokenNetwork.sol#L23) and not a `uint64`